src/test/resources/org/openmrs/uitestframework/test-ocl.properties

```diff
@@ -0,0 +1,5 @@
+webapp.url=http://localhost:8080
+login.username=haddy315
+login.password=mamanamutebi315
```

Comment:
Thanks @hadijahkyampeire  for working on this, i think we need to make this  more generic to specific crendentials , Suppose another tester come up  with different username or password, we would not have something general, can we use admin, Admin123 for password and username

Reply:
You are right @sherrif10 , I did that when we had just changed the environment to staging and didn't have uniform credentials but now we have so that will change.